env.py

```diff
@@ -11,7 +11,7 @@ class Grid(object):
     '''
 
     def __init__(self, height:int, width:int, gamma:float, white_r:float=None,
-                features_sq:List[Dict]=None, gen_features:List[List]=None, n_features:int=None,
+                features_sq:List[Dict]=None, gen_features:Union[List[List], Any]=None, n_features:int=None,
```

Comment:
change 'Any' to 'str'

Reply:
I think it is fine to leave it generalized but feel free to change it Vincent. Just trying to push for functionality and cleanliness rn